apps/researcher/src/app/[locale]/objects/[id]/page.tsx

```diff
@@ -218,9 +218,13 @@ export default async function Details({params}: Props) {
                 translationKey="locationCreated"
```

Comment:
Can you also rename the translation key to 'locationsCreated' (also in the messages files)?